src/Microsoft.AspNet.Mvc.ApiExplorer/DefaultApiDescriptionProvider.cs

```diff
@@ -6,6 +6,7 @@
 using System.Linq;
 using System.Reflection;
 using System.Threading.Tasks;
+using Microsoft.AspNet.Mvc.Description;
```

Comment:
`ApiDescriptionActionData` in https://github.com/aspnet/Mvc/blob/fix1911/src/Microsoft.AspNet.Mvc.ApiExplorer/DefaultApiDescriptionProvider.cs#L58 is using it.